rules/do-you-have-a-product-roadmap/rule.md

```diff
@@ -32,6 +34,21 @@ Simply add a list of important milestones with a checkbox next to each. Then tic
 
 Ideally, you want to make some progress on product roadmap items in every Sprint, so make sure to review your product roadmap during your regular Sprint Reviews.
 
+### Building the product roadmap
+
+It's important to think about the scope of the product when building the product roadmap so that the milestones an achievable grouping of tasks. Consider the size of your team: 
```

Comment:
Change
From 
   the milestones an achievable grouping of tasks
To
   the milestones are an achievable grouping of tasks